src/NUnitFramework/tests/nunit.framework.tests.csproj

```diff
@@ -32,6 +32,7 @@
 
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.13.0" />
+    <PackageReference Include="System.Collections.Immutable" Version="1.5.0" />
```

Comment:
I started to hit issues when building with Cake with version >= 1.6 (similar to what was documented here: https://github.com/Azure/azure-functions-vs-build-sdk/issues/353). VS builds fine with the latest version of this (1.7.1).